### PR TITLE
Add recommendation for vsClass's annotation

### PR DESCRIPTION
**Pull request by apurvabhide17**
Version(s):
OCP 4.14 - 4.18

Issue:
[OADP-4013](https://issues.redhat.com/browse/OADP-4013)

Link to docs preview:

- [Backing up persistent volumes with CSI snapshots](https://86169--ocpdocs-pr.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-backing-up-pvs-csi-doc)
- [Backing up data from one cluster and restoring it to another cluster](https://86169--ocpdocs-pr.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/oadp-advanced-topics.html#oadp-backing-and-restoring-from-cluster-to-cluster_oadp-advanced-topics)

QE review:
- [x] QE has approved this change.

Additional information:
Discussion with QE - [draft](https://docs.google.com/document/d/1iqq5-dG8nedSxG5bmudZFZxCOoyy50gu6rGJekOaMME/edit?usp=sharing) document

### CODE REVIEW

File: backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-backing-up-pvs-csi-doc.adoc [hunk 1]
```diff
@@ -34,8 +34,8 @@ driver: <csi_driver>
 deletionPolicy: <deletion_policy_type> <3>
 ----
 <1> Must be set to `true`.
-<2> Must be set to `true`.
-<3> OADP supports the `Retain` and `Delete` deletion policy types for CSI and Data Mover backup and restore. For the OADP 1.2 Data Mover, set the deletion policy type to `Retain`.
+<2> Must be set to `true`. If you are restoring this volume in another cluster with the same driver, make sure that you set the `snapshot.storage.kubernetes.io/is-default-class` parameter to `false` instead of setting it to `true`. Otherwise, the restore will partially fail.
```
Comment (PrasadJoshi12) [inline backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-backing-up-pvs-csi-doc.adoc:37]: I think we should change this field to optional.

Comment (apurvabhide17) [reply backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-backing-up-pvs-csi-doc.adoc]: done, thanks @PrasadJoshi12

**Comment by mateusoliveira43** [comment]
@apurvabhide17 can you add doc preview link in PR description?

File: backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-backing-up-pvs-csi-doc.adoc [hunk 2]
```diff
@@ -34,8 +34,8 @@ driver: <csi_driver>
 deletionPolicy: <deletion_policy_type> <3>
 ----
 <1> Must be set to `true`.
-<2> Must be set to `true`.
-<3> OADP supports the `Retain` and `Delete` deletion policy types for CSI and Data Mover backup and restore. For the OADP 1.2 Data Mover, set the deletion policy type to `Retain`.
+<2> Optional: Must be set to `true`. If you are restoring this volume in another cluster with the same driver, make sure that you set the `snapshot.storage.kubernetes.io/is-default-class` parameter to `false` instead of setting it to `true`. Otherwise, the restore will partially fail.
```
Comment (mateusoliveira43) [inline backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-backing-up-pvs-csi-doc.adoc:37]: I think it is confusing to say optional and that it must be true. Would be better to remove that part and just have the `If you are restoring this volume ...` part?

Comment (apurvabhide17) [reply backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-backing-up-pvs-csi-doc.adoc]: Done, thanks

**Comment by apurvabhide17** [comment]
> @apurvabhide17 can you add doc preview link in PR description?

@mateusoliveira43, I've updated the description of the PR with preview links. Let me know if there if it needs any other update.
Thanks!

**Review by GroceryBoyJr — COMMENTED** [review]
LGTM

**Review by mateusoliveira43 — APPROVED** [review]

**Comment by apurvabhide17** [comment]
Thank you, @GroceryBoyJr, @PrasadJoshi12,  and @mateusoliveira43 for reviewing the changes.

**Pull request merged**